packages/plugin-sample-api/src/models/Type.ts

```
@@ -0,0 +1,39 @@
+        public static async createType(doc) {
+            return models.Types.create({ ...doc });
+        }
+        // remove type
+        public static async removeType(_id: string) {
+            return models.Types.deleteOne({ _id });
+        }
+
+        public static async updateType(_id: string, doc) {
+            return models.Types.updateOne({ _id }, { $set: { ...doc } });
+        }
+    }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)